Sitemap/Url/GoogleImage.php

```diff
@@ -49,39 +49,116 @@ class GoogleImage
     /**
      * create a GoogleImage for your GoogleImageUrl
      *
-     * @param string      $loc
-     * @param string|null $caption      [optional]
-     * @param string|null $geo_location [optional]
-     * @param string|null $title        [optional]
-     * @param string|null $license      [optional]
+     * @param string      $location
+     * @param string|null $caption     [optional]
+     * @param string|null $geoLocation [optional]
+     * @param string|null $title       [optional]
+     * @param string|null $license     [optional]
      */
-    public function __construct($loc, $caption = null, $geo_location = null, $title = null, $license = null)
+    public function __construct($location, $caption = null, $geoLocation = null, $title = null, $license = null)
     {
-        $this->setLoc($loc);
+        $this->setLocation($location);
         $this->setCaption($caption);
-        $this->setGeoLocation($geo_location);
+        $this->setGeoLocation($geoLocation);
         $this->setTitle($title);
         $this->setLicense($license);
     }
 
+    public function __get(string $name)
```

Comment:
Unfortunately, adding type hint will be a huge BC Break.
I'm planning to add type hint for the next `3.0.0` major version.
To be honest, I used a phpstorm template to generate these magic methods.
I considered this particular type hint "safe" because PHP only supports string type names, and `2.x`, but you're right, I will remove it here for the sake of simplicity !